modules/imgproc/src/imgwarp.cpp

```diff
@@ -50,10 +50,10 @@
 #include "precomp.hpp"
 #include "opencl_kernels_imgproc.hpp"
 #include "hal_replacement.hpp"
-
+#include "opencv2/core/hal/intrin.hpp"
 #include "opencv2/core/openvx/ovx_defs.hpp"
 #include "imgwarp.hpp"
-
+#include <iostream>
```

Comment:
Is it required?

Reply:
Yep, It was the debug use.
I reverted. Thank you